src/ffi/wasm.rs

```diff
@@ -32,13 +29,15 @@ type Result<T> = std::result::Result<T, JsValue>;
 ///
 /// # Safety
 ///
-/// - If the same seed is used twice, the blinded result WILL be the same
-pub fn blind(message: &[u8], seed: &[u8]) -> BlindedMessage {
+/// - If the same seed and message is used twice, the blinded result WILL be the same.
+pub fn blind_msg(message: &[u8], seed: &[u8]) -> BlindedMessage {
     // convert the seed to randomness
-    let mut rng = get_rng(seed);
+    let mut rng = get_rng(&[message, seed]);
 
     // blind the message with this randomness
-    let (blinding_factor, blinded_message) = POPRF::blind_msg(message, &mut rng);
+    // DO NOT MERGE(victor): Should the error here be handled or is it OK to panic? I can't think
```

Comment:
Seems better to just handle the error? Is there a reason not to handle it?

Reply:
@mstraka100 I changed this to handle the error